bears/r/FormatRBear.py

```diff
@@ -14,8 +19,62 @@ class FormatRBear:
     LANGUAGES = "R"
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
+    def create_arguments(filename, file, config_file,
+                         r_keep_comments: bool=True,
+                         r_keep_blank_lines: bool=True,
+                         r_braces_on_next_line: bool=False,
+                         r_use_arrows: bool=False,
+                         tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
+                         r_max_expression_length: int=20):
+        """
+        :param r_keep_comments:
+            Determines whether comments are kept or not.
+        :param r_keep_blank_lines:
+            Determines whether blank lines are kept or not.
+        :param r_braces_on_next_line:
+            Determines whether a brace should be placed on the next line.
+
+            Example:
+            If ``True``,
+            ```
+            if (...) {
+            ```
+            changes to
+            ```
+            if (...)
+            {
+            ```
+            If ``False`` the brace is placed on the same line.
+        :param r_use_arrows:
+            Determines if either the assign operator ``=`` or the arrow ``<-``
+            should be used.
+
+            Example: If  ``True``, ``x = 1`` changes to ``x <- 1``.
+        :param tab_width:
+            Number of spaces for indentation.
+        :param r_max_expression_length:
+            Maximum number of characters for an expression.
+
+            Example: If ``20`` then
+            ``1 + 1 + 1 + 1 + 1 + 1 + 1`` changes to
+            ``1 + 1 + 1 + 1 + 1 + 1 +``
+            ``    1``
```

Comment:
this one needs also triple quotes, otherwise you won't see the \n you want to point out^^
(using here ''' instead so github markdown doesn't interprete it) -->
```
Example: If ``20`` then
'''
1 + 1 + 1+ 1 ...
'''
changes to
'''
1 + 1+ 1 + 1 +
    1
'''
```